documentation/frontend/guides/quickstart.md

```diff
@@ -38,7 +38,19 @@ you need to run this.
    $ just node-install
    ```
 
-4. To bring up the frontend, we have another `just` recipe. We have `just`
+4. If you encounter errors after this, you may need to compile just the Openverse ESLint plugin.
+
+   ```console
+   $ pnpm --filter='@openverse/eslint-plugin' build
+   ```
+
+   Or
+
+   ```console
+   $ pnpm -F '@openverse/eslint-plugin' build
+   ```
```

Comment:
This seems an unnecessary repetition of the previous command. We could omit this.